observer/probers/prober.go

```diff
@@ -49,11 +60,17 @@ type Configurer interface {
 // the `MonConf`.
 type Settings map[string]interface{}
 
+// NormalizedKind normalizes the input string by stripping spaces and
+// transforming it into lowercase
+func NormalizedKind(kind string) string {
```

Comment:
two nits: since this is no longer being called in mon_conf.go, it doesn't need to be Public anymore; and we tend to name functions by what they _do_ (see `Register()` below), so this would usually be named `normalizeKind`, rather than `normalizedKind`.

Reply:
Right, totally forgot about making that private, thanks!